lua/gatekeeper/dylib.lua

```diff
@@ -81,6 +81,28 @@ int gk_flush_flow_table(const char *src_prefix,
 
 c = ffi.C
 
+local function fib_action_to_str_map(fib_action)
```

Comment:
Given that this is a function instead of a table, I'd drop the suffix "_map" from the name, that is, I'd call this function `fib_action_to_str()`.